main/ertp/guide/README.md

```diff
@@ -26,87 +26,14 @@ post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
-purse can only ever hold BaytownBucks.
-
-Let's distribute the BaytownBucks to members of the community. To send
-money in ERTP, we withdraw [`payments`](./mint.html#payments) from purses.
-
-```js
-const paymentForAlice = purse.withdraw(10, `alice's community money`);
-```
-
-Like our purse, this payment contains BaytownBucks, but unlike purses,
-payments are used to represent tokens in transit. A payment can be
-sent to someone else, a purse never should be.
-
-Now let's send the payment to Alice as message:
-
-```js
-alice.receivePayment(paymentForAlice);
-```
-
-This may seem strange, but ERTP is built on top of [an
-infrastructure](https://github.com/Agoric/SwingSet) in which
-everything is an object. In this example, we have a reference to the
-object `alice`, and can call her `receivePayment` to ask her to
-receive this payment. Alice's methods are entirely up to her, and are
-not part of ERTP.
-
-## Security Properties
-
-How does Alice know that she got paid real money? She could have been
-sent fake money, or she could have been sent money that was
-[double-spent](https://en.wikipedia.org/wiki/Double-spending).
-
-When alice receives an alleged payment, she can call a method to know
-that the alleged payment was valid, and get a new payment that is
-exclusively hers:
-
-```js
-const myExclusivePayment = BaytownBucksIssuer.claimAll(allegedPayment);
-```
-
-The BaytownBucksIssuer is associated with the BaytownBucksMint, but
-the issuer is the public-facing version that is accessible to anyone.
-By holding the reference to a mint, you can mint more tokens. By
-holding a reference to the issuer for a mint, you can check that a
-payment is valid and exclusively claim it in a new payment to yourself.
-
-That's the basic use case for a fungible token. `makeMint` in
-[issuers.js](core/issuers.js) takes
-in an optional configuration that allows for many more possibilities.
-
-## Pixel Demo
-
-For a demo on how ERTP can be used, check out our [Pixel Demo](/documentation/getting-started/pixel-demo.html#installation).
-
-### A preemption hierarchy of rights
 
-All of the pixels (individual squares) on the demo webpage are owned by the gallery, the code that administrates the pixels. The gallery has the power to revoke the pixels from any user at any time. When a user calls `gallery!tapFaucet()`, the gallery takes the least recently colored pixel from its current holder (if anyone holds it) and gives it to the user in the form of an ERTP payment. The gallery is able to revoke the pixels held by users because the pixel demo creates a customized version of ERTP in which rights are hierarchical. Hierarchical rights are familiar in property. For instance, a property owner might lease an apartment to a tenant, who might in turn, lease it to a subtenant. Anyone higher in the hierarchy can take away access from anyone lower in the hierarchy, and give it to someone else.
 
 ## More ERTP resources
```

Comment:
While we're at it, let's remove the items in this section. They were placeholders when we didn't have any content.